### PR TITLE
Parameterize host image and use Cirros or Ubuntu

### PR DESCRIPTION
**Pull request by maelk**

### CODE REVIEW

**Comment by maelk** [comment]
I'm putting a PR for shellcheck fixes together

**Comment by hardys** [comment]
AFAICS this removes the centos option completely?  I'm wondering if we should just include cirros by default, and document how to override the config for other distros in the config_$user.sh?

Alternatively it'd be nice to leave the centos option as some of us are actively testing things on Centos

**Comment by maelk** [comment]
I have added the Centos option. I will submit a PR to change the try-it.html

File: config_example.sh [hunk 1]
```diff
@@ -83,3 +83,31 @@
...
+#
+#export IMAGE_OS="Cirros"
+
+# Image for target hosts deployment
+#
+#export IMAGE_NAME="CentOS-7-x86_64-GenericCloud-1907.qcow2"
+
+# Location of the image to download
+#
+#export IMAGE_LOCATION="http://cloud.centos.org/centos/7/images"
```
Comment (hardys) [inline config_example.sh]: We should probably show the sample IMAGE_NAME/IMAGE_LOCATION as the defaults, e.g now cirros?  That would also be more consistent with the IMAGE_OS above and the IMAGE_USERNAME

Comment (maelk) [reply config_example.sh]: True, will fix! thank you

**Comment by maelk** [comment]
Here is the documentation update : https://github.com/metal3-io/metal3-io.github.io/pull/104

**Comment by maelk** [comment]
The ubuntu test was a false positive. Re-running it

**Comment by maelk** [comment]
/hold 
CI Metal3-dev-env broken on Ubuntu

**Comment by maelk** [comment]
/hold cancel
CI fixed

**Pull request merged**